template.yaml

```diff
@@ -19,10 +19,9 @@ Globals:
     Environment:
       Variables:
         ALLOWED_ORIGIN: '*'
-        CRISTIN_API_HOST: 'api.cristin.no'
-        DOMAIN_NAME: 'api.dev.nva.aws.unit.no' # TODO: NP-2280: Dynamic from CustomDomain
-        # TODO: NP-2280: Replace with CustomDomainBasePath and call it PROJECTS_BASE_PATH
-        BASE_URL: 'https://api.dev.nva.aws.unit.no/project'
+        CRISTIN_API_HOST: !Ref CristinApiHost
+        DOMAIN_NAME: !Ref CustomDomain
```

Comment:
Do we need separate DOMAIN_NAME and BASE_PATH? Can we do !Sub 'https://${CustomDomain}/${CustomDomainBasePath}/'?

Reply:
As far as I remember we discussed it and rather agreed on having them as separate. We should ask @brinxmat about it.